configure.ac

```diff
@@ -1484,8 +1484,25 @@ AC_ARG_ENABLE([armasm],
     [ ENABLED_ARMASM=$enableval ],
     [ ENABLED_ARMASM=no ]
     )
-if test "$ENABLED_ARMASM" = "yes" && test "$ENABLED_ASM" = "yes"
-then
+if test "$ENABLED_ARMASM" != "no" && test "$ENABLED_ASM" = "yes"
```

Comment:
Can you update the help to reflect this new option? Also the comment above says “AES ARM”, can you fix that?

Reply:
Done.
Let me know if there is anything else.